packages/domainproperties/src/components/assay/AssayPropertiesInput.tsx

```diff
@@ -16,22 +16,51 @@ interface AssayPropertiesInputProps {
 }
 
 export class AssayPropertiesInput extends React.PureComponent<AssayPropertiesInputProps, any> {
+
+    getSplitSentence = (label: string, lastWord: boolean): string => {
```

Comment:
These looks like a function that should be out in an actions.ts file and not part of a component. I think that would make it easier to add jest tests for it. It doesn't use any state/props and only relies on incoming params.